configs/configupgrade/analysis_expr.go

```diff
@@ -128,6 +128,11 @@ func (d analysisData) GetModuleInstance(addrs.ModuleCallInstance, tfdiags.Source
 }
 
 func (d analysisData) GetModuleInstanceOutput(addrs.ModuleCallOutput, tfdiags.SourceRange) (cty.Value, tfdiags.Diagnostics) {
+	// This implementation doesn't use this.
```

Comment:
Is this comment denoting that this is dead code?

Reply:
Yes, this function is required so that the interface is implemented, but is not used - I will add to the comment!